buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -661,6 +662,25 @@ def retry_and_delay(command, numRetries = 3, waitTime = 60, units = 'SECONDS') {
 }
 
 def setup_pull_request() {
+    // Parse Github trigger comments
+    // For example:
+    /* This is great
+       Jenkins test extended <platform>
+       It was tested*/
+
+    def PARSED_BY_NEWLINE_COMMENT = params.ghprbCommentBody.split(/\\r?\\n/)
+    for (COMMENT in PARSED_BY_NEWLINE_COMMENT) {
+        echo "Trigger Comment: '${COMMENT}'"
```

Comment:
This debug output should be removed, or only used when there is a match. Otherwise it's printing every line of every comment.